crates/core/src/models/event.rs

```diff
@@ -87,3 +85,75 @@ impl ConvertibleStarknetEvent for StarknetEvent {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::client::tests::init_client;
+    use crate::mock::mock_starknet::fixtures;
+
+    #[test]
+    fn test_to_eth_log_log3() {
+        // Given
+        let event: Event = serde_json::from_str(include_str!("test_data/conversion/starknet/event_log3.json")).unwrap();
+        let starknet_event = StarknetEvent::new(event);
+
+        let fixtures = fixtures(vec![]);
+        let client = init_client(Some(fixtures));
+
+        // When
+        let eth_log = starknet_event.to_eth_log(&client, None, None, None, None, None).unwrap();
```

Comment:
I can add a small test on block numbers, hash, index but it actually doesn't get used in the function at all, just gets passed along, see [here](https://github.com/kkrt-labs/kakarot-rpc/blob/main/crates/core/src/models/event.rs#L77).

Reply:
Yeah, that's what I was thinking, I guess it would only be a precautionary measure for for some reason we eventually have the logic changed, but I guess not having is also fine

Reply:
I think since now we are on it, we could add a simple one as for later this will out of our awareness and for prod maybe its better to be safer?